### PR TITLE
activity_settings.xml

### PR DESCRIPTION
**Pull request by kulloveth**
This xml file have been changed to use constraintLayout

### CODE REVIEW

**Comment by kulloveth** [comment]
@ujjwalagrawal17  kindly review changes made

File: app/src/main/res/layout/activity_settings.xml [hunk 1]
```diff
@@ -18,164 +25,199 @@
...
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/padding_8dp"
-        android:orientation="vertical">
+        android:layout_marginTop="@dimen/padding_8dp"
+        android:fillViewport="true"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/appBarLayout2">
 
-        <androidx.cardview.widget.CardView xmlns:card_view="http://schemas.android.com/tools"
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/activity_settings.xml]: avoid having nested constraint Layouts. Other things are working good.

Comment (ujjwalagrawal17) [reply app/src/main/res/layout/activity_settings.xml]: @kulloveth can you remove this?

**Comment by kulloveth** [comment]
@ujjwalagrawal17  i thought about that too but you know scrollView can have only one direct child plus if I use it as the parent layout the toolbar will be scrolling. Kindly suggest

**Comment by kulloveth** [comment]
> @ujjwalagrawal17 i thought about that too but you know scrollView can have only one direct child plus if I use it as the parent layout the toolbar will be scrolling. Kindly suggest. is it a good practice for the toolbar to be scrolling?

I used CardView instead, what do you think?

**Comment by ujjwalagrawal17** [comment]
Ohhhh yes. ScrollView can have only one child. Let's use a nested constraint layout and not put toolbar inside scroll view..

**Comment by kulloveth** [comment]
@ujjwalagrawal17  can you check if the cardView I used will do the job

**Comment by kulloveth** [comment]
@ujjwalagrawal17 am still waiting to know if you are good with this

**Comment by ujjwalagrawal17** [comment]
I will review it after 5-6 hours.

**Pull request merged**